### ISSUE TITLE
node-forge npm critical security error

### ISSUE THREAD

**Issue by farisT** [digitalbazaar/forge#926]
Hi,

When installing packages dependent on node-forge I seem to be getting npm audit critical errors for the set() in debug.js
 
![Screenshot 2021-11-23 at 14 03 51](https://user-images.githubusercontent.com/33259789/143029030-27ad919b-506a-4782-8b44-bab718e63b2a.png)

Looking around I saw that someone made a fix for this - https://github.com/418sec/forge/pull/ is it possible to consider this for a current fix - or does more work need to be done?

**Comment by farisT** [comment]
Update: I've seen that changes merged to master have not been deployed yet which actually fix this issue (see the fix here merged to master already https://github.com/digitalbazaar/forge/commit/51228083550dde97701ac8e06c629a5184117562). When can we see version 0.11.0 published?

**Comment by silkfire** [comment]
Agree, I can see that this repository is actively being pushed to but the last release was made in September 2020?

**Comment by farisT** [comment]
Right now the only wait to solve this is to point directly to the commit on master instead of version 0.10.0 and use `npm-force-resolutions` package.

```
  "resolutions": {
    "node-forge": "https://github.com/digitalbazaar/forge.git#51228083550dde97701ac8e06c629a5184117562"
  }
```

**Comment by davidlehn** [comment]
Fixed in 1.0.0.
